torch/onnx/_internal/exporter/_torchlib/_torchlib_registry.py

```diff
@@ -5,37 +5,96 @@
 from __future__ import annotations
 
 
-__all__ = ["registry", "onnx_impl"]
+__all__ = ["onnx_impl", "get_torchlib_ops"]
 
-import collections
-from typing import Callable, TypeVar
+import logging
+from typing import Any, Callable, Sequence, TypeVar
 
+import onnxscript
 
-_T = TypeVar("_T", bound=Callable)
-
+import torch
+from torch.onnx._internal.exporter import _registration, _schemas
+from torch.onnx._internal.exporter._torchlib import _constants
 
-class Registry(collections.UserDict[Callable, list[Callable]]):
```

Comment:
What will be the replacement?

Reply:
The replacement is the _registry list I defined somewhere in this file. Since this registry is only used for collecting op implementations, I removed all complex data structures on this file. The list will be consumed by the registry in the exporter at the export call.

Reply:
How can we define a custom op then?

Reply:
The custom op is handled by the exporter registry, and so the api doesn’t change